### PR TITLE
Add `CaptureAs` and `ReplaceCssVariables`

### LINKED ISSUE #123: Option to Store Computed Colors on `nodes` and `links`
## Elevator Pitch

* Like with `NodeSelection` and `LinkSelection`, add an option to `ContinuousColor` and `OrdinalColor` to store the value in a column.

## Motivation

* We want to be able to color `links` based on the color of their `source` or `target`, but when the `nodes` are colored using `ContinuousColor`, it is not possible to reference a `node`'s color.
* This is motivated from the concept of groups in a graph, where we not only want to color the `nodes` based on their group assignment, but also the `links` that connect between two `nodes` in the same group.

## Design Ideas

* Add a `column_name` trait to `ContinuousColor` and `OrdinalColor` and store the computed color under that name,
* As with `NodeSelection` and `LinkSelection`, the `default_value = None` and in which case, nothing is stored.
* We may want to check that the user does not specify `__indexColor` as that seems to be a reserved property in `forcegraph`.

### PR DESCRIPTION
**Pull request by nrbgt**
## Checklist

- [x] the PR target is correct
  - `main` for releases, docs-only fixes and post-release hot-fixes
  - [x] `dev` for everything else
- [x] ran `doit lint` locally

## References

- fixes #123
- alternative to #124

## Code changes

- [x] bump some versions
- [x] add wrapper base class
- [x] add `CaptureAs`
- [x] add `ReplaceCssVariables`
- [x] add simple example
- [x] update existing examples relying on `var(..)`   
- [x] update tests

## User-facing changes

- users can capture any calculated value

| note | screen |
|-|-|
| replacing CSS vars and capturing (multiple) calculated values | image |

## Backwards-incompatible changes

- the behavior introduced in `0.3.8` for replacing `var(--{variable name})` is removed, favoring `ReplaceCssVariables`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
 # Changelog
 
-## `0.3.9`
+## `0.4.0`
 
-### `ipyforcegraph 0.3.9`
+### `ipyforcegraph 0.4.0`
 
-> TBD
+- adds `CaptureAs`, which can wrap a `Nunjucks` or `ContinuousColor` to store the
```
Comment (sanbales) [inline CHANGELOG.md:7]: It'd be dumb to do, but we can also wrap `Column` since it is a `DynamicValue`?

**Review by sanbales — APPROVED** [review]
Looks good, sir!

File: CHANGELOG.md (hunk 1 above)
Comment (nrbgt) [reply CHANGELOG.md:7]: yeah, might already work. would add test.

Comment (nrbgt) [reply CHANGELOG.md:7]: actually, the `Color` stuff is _already_ a `Column` subclass... maybe i just add to the example.

Comment (nrbgt) [reply CHANGELOG.md]: is now under test :test_tube: , updated text

**Pull request merged**